tests/testthat/helper-tests.R

```diff
@@ -0,0 +1,17 @@
+try(
+  {
+    # Snapshots
+    rifampicin_snap <- Snapshot$new(test_path("data/Rifampicin-Model.json"))
+
+    # Compounds
+    rifampicin <- compound(test_path("data/Rifampicin-Model.json"))
+    midazolam <- compound(test_path("data/Midazolam-Model.json"))
+    levonorgestrel <- compound(test_path("data/Levonorgestrel-Model.json"))
+    itraconazole <- compound(test_path("data/Itraconazole-Model.json"))
+    itraconazole80 <- compound(test_path("data/Itraconazole-Model-80.json"))
+
+    # DDI
+    levo_itra_ddi <- import_ddi(test_path("data/Levonorgestrel-Itraconazole-DDI.json"))
+  },
+  silent = T
+)
```

Comment:
What is the rational of wrapping evrything into `try()`? If one line fails, remaining lines won't be executed

Reply:
testthat helpers are evaluated when we load the package with `devtools::load_all()`. If one of these object is temporary broken, it would be impossible to load the package before fixing it.
So here, when loading the package in dev environment, we try to create all these objects. If we can't, it doesn't matter, these failures will be discovered by test anyway.

Reply:
Got it, thanks for explaining the rationale behind the chosen approach -makes sense in the dev environment.